src/services/axios.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
```

Comment:
maybe call the file `axios-wrapper` it would be less confusing (it is not the real exios)